modules/3d/test/test_pointcloud.cpp

```diff
@@ -0,0 +1,148 @@
+#include <opencv2/core.hpp>
+#include <iostream>
+#include <vector>
+#include <string>
+#include <cstdio>
+
+#include "test_precomp.hpp"
+#include "opencv2/ts.hpp"
+
+namespace opencv_test { namespace {
+
+TEST(obj_test, point_cloud_load_tests)
+{
+    std::vector<cv::Point3f> points_1 = {
+        {-5.93915f, -0.13257f, 2.55837f},
+        {-5.93915f, 1.86743f, 2.55837f},
+        {-5.93915f, -0.13257f, -1.16339f},
+        {-5.93915f, 1.86743f, -1.16339f},
+        {0.399941f, -0.13257f, 2.55837f},
+        {0.399941f, 1.86743f, 2.55837f},
+        {0.399941f, -0.13257f, -1.16339f},
+        {0.399941f, 1.86743f, -1.16339f}};
+    std::vector<cv::Point3f> normals_1 = {
+        {-1.0000f, 0.0000f, 0.0000f},
+        {0.0000f, 0.0000f, -1.0000f},
+        {1.0000f, 0.0000f, 0.0000f},
+        {0.0000f, 0.0000f, 1.0000f},
+        {0.0000f, -1.0000f, 0.0000f},
+        {0.0000f, 1.0000f, 0.0000f}};
+
+    std::vector<cv::Point3f> points;
+    std::vector<cv::Point3f> normals;
+
+    std::string folder = string(cvtest::TS::ptr()->get_data_path()) + "pointcloudio/";
+    std::string original_path = folder + "orig.obj";
+
+    cv::loadPointCloud(original_path, points, normals);
+
+    EXPECT_TRUE(points_1==points);
+    EXPECT_TRUE(normals_1==normals);
+}
+
+TEST(obj_test, point_cloud_load_no_norms_tests)
```

Comment:
the word "_tests" is redundant in test names. I propose to remove it everywhere here.